modules/plugin/jdbc/jdbc-db2/src/main/java/org/geotools/data/db2/DB2FilterToSQL.java

```diff
@@ -130,6 +130,29 @@
     // Class to convert geometry value into a Well-known Text string
     private static WKTWriter wktWriter = new WKTWriter();
 
+    /**
+     * Conversion factor from common units to meter
+     */
+    private static final Map<String, Double> UNITS_MAP = new HashMap<String, Double>() {
+        {
+            put("kilometers", 1000.0);
+            put("kilometer", 1000.0);
+			put("meters", 1.0);
```

Comment:
This line uses tabs, please switch to spaces (yes, there is old code that's not properly formatted, for newer code please follow the coding conventions, and no, do not mass reformat files)